tensorboard/uploader/uploader.py

```diff
@@ -208,15 +218,6 @@ def start_uploading(self):
             self._upload_once()
             if self._one_shot:
                 break
-        if self._one_shot and not self._tracker.has_data():
-            print(
-                "Tensorboard was run in `one_shot` mode, but did not detect "
```

Comment:
"Tensorboard" -> "TensorBoard" (although I guess this was already there and is just being deleted? Might want to make sure we don't have other instances of this).